src/Terrain/MapMaster.java

```diff
@@ -65,14 +64,14 @@ public MapMaster(CommandingOfficers.Commander[] COs, MapInfo mapInfo)
           final TerrainType terrainType = map[x][y].getEnvironment().terrainType;
           if( terrainType == TerrainType.HEADQUARTERS )
           {
-            COs[co].HQLocations.add(new XYCoord(x, y));
+            propertyOwners[co].HQLocations.add(new XYCoord(x, y));
             hasHQ = true;
           }
           else if( terrainType == TerrainType.LAB )
           {
             hasLab = true;
           }
-          location.setOwner(COs[co]);
+          location.setOwner(propertyOwners[co].cos[0]);
```

Comment:
No support for assigning properties to multiple COs in the same army, eh?

Reply:
That's only relevant under Team Merge, which has its own support for that very concept.
The other tag variants always have Lead CO == Owner of All Properties.